src/login/logind-dbus.c

```diff
@@ -311,7 +311,7 @@ static int property_get_inhibited(
         assert(bus);
         assert(reply);
 
-        w = manager_inhibit_what(m, streq(property, "BlockInhibited") ? INHIBIT_BLOCK : INHIBIT_DELAY);
+        w = manager_inhibit_what(m, /* block= */ streq(property, "BlockInhibited"));
```

Comment:
this doesn't look right. we should have three props now. for the the three relevant types of inhibition: block, block-weak, delay.